rector.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\ClassConstFetch\RenameClassConstFetchRector;
+use Rector\Renaming\ValueObject\RenameClassConstFetch;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/examples',
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/tools',
+    ]);
+
+    /**
+     * ReadPreference::RP_* constants are deprecated in favor of the string constants
```

Comment:
Thinking about this some more. I know the change was helpful for implementing the PR, but does this make sense as a permanent change in the Rector configuration?

The constants are technically only interchangeable in the ReadPreference constructor, and I don't see anything here that limits the replacement to that context. Hypothetically, if someone else was using the original constants with an integer type hint, applying this rule could yield invalid code, no?

Reply:
I'll track the rector config in this PR description and remove rector from this PR.